java/build.sh

```diff
@@ -0,0 +1,10 @@
+VERSION="25.02"
```

Comment:
Make sure this version gets updated when `ci/release/update_versions.sh` is called.